hrm-domain/scheduled-tasks/hrm-data-pull/context.ts

```diff
@@ -17,9 +17,9 @@
 import { getSsmParameter } from '@tech-matters/ssm-cache';
 import { rulesMap } from '@tech-matters/hrm-core/permissions/rulesMap';
 
-const sanitizeEnv = (env: string) => (env === 'local' ? 'development' : env);
+// const sanitizeEnv = (env: string) => (env === 'local' ? 'development' : env);
 
-let hrmEnv = sanitizeEnv(process.env.NODE_ENV ?? 'development');
+let hrmEnv = process.env.NODE_ENV ?? 'development';
 let shortCode = process.env.HL ?? (hrmEnv === 'development' ? 'AS' : 'CA');
```

Comment:
Also, the fallback here doesn't make too much sense now, since it's not only `CA` that uses data exports.
I think we should remove the fallback, and just stay with `let shortCode = process.env.HL`.